index.html

```diff
@@ -31,23 +37,26 @@ <h2>I'am a Software Developer</h2>
              you see and have a project you need coded, 
              don't hestiate to contact me.
         </p>
-        
+        </div>
+        <div class="intro-icons">
        <ul>
         <li><img src="Images/Icons/NormalButton/Tertiary/Icons/github-icon.png" alt="soclai media icons"></li>
         <li><img src="Images/Icons/Icon-Linkedin.png" alt="soclai media icons"></li>
         <li><img src="Images/Icons/hand-icon.png" alt="soclai media icons"></li>
         <li><img src="Images/Icons/NormalButton/Tertiary/Icons/twitter-icon.png" alt="soclai media icons"></li>
         <li><img src="Images/Icons/medium-icon.png" alt="soclai media icons"></li>
```

Comment:
All the social icons should be links, so visitors can easily find you on other social platforms.